mautic/mautic
# [UI] Notification panel

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#191 
| Issue(s) addressed                     | Fixes #13141
 



## Description

The updated notification panel now differentiates between alert types by introducing distinct visual cues. The most recent notification, marked as an info alert, features a blue border and a blue information icon, distinguishing it from the default alerts below. Previously, all notifications appeared identical, which could lead to confusion. 

This enhancement allows users to quickly identify the type of alert, improving the clarity and usability of the notification system. Users benefit from this change by being able to prioritize and respond to notifications more effectively.

Also, now time use a totally humanized labeling system, telling how old is the notification in the format "X time ago".




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/0c754606-af8b-4cd1-aaa4-0732b5112dc2)                                      | ![image](https://github.com/user-attachments/assets/2bd721e5-0f5e-4ef5-97f2-cac40a86aecd)






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Contacts > Page actions > Export to CSV to trigger a notification
3. Reload the page
4. Open the Notifications panel
5. See the new style
